### PR TITLE
Gas simulations refactor

### PR DESCRIPTION
**Pull request by akaija**
Created separate data tables for each simulation type.

### CODE REVIEW

File: htsohm/db/material.py [hunk 1]
```diff
@@ -16,120 +12,24 @@ class Material(Base):
...
-    vf_helium_void_fraction = Column(Float)                   # dimm.
-
-    # bins
-    gas_adsorption_bin = Column(Integer)                     # dimm.
-    surface_area_bin = Column(Integer)                        # dimm.
-    void_fraction_bin = Column(Integer)                       # dimm.
+    
+    id         = Column(Integer, primary_key=True)
+    run_id     = Column(String(50)) 
+    uuid       = Column(String(40))
```
Comment (paulboone) [inline htsohm/db/material.py]: get rid of uuid?

File: htsohm/db/material.py [hunk 2]
```diff
@@ -143,155 +43,22 @@ def __init__(self, run_id=None, ):
...
+#        Args:
+#            self (class): row in material table.
+#
+#        Returns:
+#            The bin corresponding to a material's gas loading, void
+#            fraction, and surface area data and their postion in this three-
+#            dimension parameter-space.
+#
+#        """
+#        binning = [*[e[0] for e in session.query(GasLoading.bin).filter(GasLoading.material_id == self.id).all()]
```
Comment (paulboone) [inline htsohm/db/material.py]: to be changed after conversation about how bins will work

File: htsohm/pseudomaterial_generator/random.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+        If the minimum number density results in a unit cell with less than 2
+        atom-sites with the given lattice constants, a minimum number density
+        of TWO ATOM SITES PER UNIT CELL is imposed.
+    
+    """
+    min_ND = number_density_limits[0]
+    max_ND = number_density_limits[1]
+    v = structure.volume()
+    min_atoms = int(min_ND * v)
```
Comment (paulboone) [inline htsohm/pseudomaterial_generator/random.py]: use math.ceil instead of int

File: htsohm/pseudomaterial_generator/random.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    max_atoms = int(max_ND * v)
+    if min_atoms < 2:
+        min_atoms = int(2)
```
Comment (paulboone) [inline htsohm/pseudomaterial_generator/random.py]: min_atoms = 2

should be fine

File: htsohm/pseudomaterial_generator/random.py [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    sigma_limits            = config["sigma_limits"]
+    max_charge              = config["charge_limit"]
+
+    ########################################################################
+    # create database row
+    material = Material(run_id)
+    material.seed = seed
+
+    # initialize incrementable value for random number generation
+    seed
```
Comment (paulboone) [inline htsohm/pseudomaterial_generator/random.py]: should this be doing something else here?

File: htsohm/db/material.py (hunk 1 above)
Comment (akaija) [reply htsohm/db/material.py]: commit 35da8a7b35cc4eef63fdee10fdcfdcd3640c512f
Author: akaija <a.r.kaija@gmail.com>
Date:   Wed Jun 20 15:59:12 2018 -0400

    removed material UUIDs, using seed integer where necessary

File: htsohm/db/material.py (hunk 2 above)
Comment (akaija) [reply htsohm/db/material.py]: commit 823f9a845e6a8505742e12ac4b345d3c09867c83
Author: akaija <a.r.kaija@gmail.com>
Date:   Wed Jun 20 16:01:14 2018 -0400

    removed commented-out code

File: htsohm/pseudomaterial_generator/random.py (hunk 3 above)
Comment (akaija) [reply htsohm/pseudomaterial_generator/random.py]: commit 71417d407148f441b739ed1a1e9c497b4ebf5948
Author: akaija <a.r.kaija@gmail.com>
Date:   Wed Jun 20 16:09:46 2018 -0400

    fixed random number density function

File: htsohm/pseudomaterial_generator/random.py (hunk 4 above)
Comment (akaija) [reply htsohm/pseudomaterial_generator/random.py]: commit 71417d407148f441b739ed1a1e9c497b4ebf5948
Author: akaija <a.r.kaija@gmail.com>
Date:   Wed Jun 20 16:09:46 2018 -0400

    fixed random number density function

File: htsohm/pseudomaterial_generator/random.py (hunk 5 above)
Comment (akaija) [reply htsohm/pseudomaterial_generator/random.py]: this is leftover code from when i was initializing the seed value within this function, i'd moved that step up one level, but i just added comments so it's more clear.

see:

commit 4d648c538c339be56486584051e7a42f08a77cc5
Author: akaija <a.r.kaija@gmail.com>
Date:   Wed Jun 20 16:12:01 2018 -0400

    added comments explaining where seed is initialized

**Pull request merged**